packages/blockchain-wallet-v4-frontend/src/modals/BuySell/CryptoSelection/index.tsx

```diff
@@ -12,16 +13,18 @@ import Loading from '../template.loading'
 import { getData } from './selectors'
 import Success from './template.success'
 
-class CryptoSelection extends PureComponent<Props> {
+class CryptoSelection extends React.Component<Props> {
   componentDidMount() {
     if (this.props.fiatCurrency && !Remote.Success.is(this.props.data)) {
       this.props.buySellActions.fetchPairs({ currency: this.props.fiatCurrency })
       this.props.buySellActions.fetchFiatEligible(this.props.fiatCurrency)
       this.props.buySellActions.fetchSDDEligibility()
-      this.props.buySellActions.fetchOrders()
+      this.props.buySellActions.fetchBSOrders()
     }
   }
 
+  shouldComponentUpdate = (nextProps) => !equals(this.props, nextProps)
```

Comment:
Here you can keep this one, because the component is already a class component, but it would be nice if you transformed this one into a FC